wgpu-native/src/hub.rs

```diff
@@ -207,3 +207,25 @@ pub struct Hub {
 lazy_static! {
     pub static ref HUB: Hub = Hub::default();
 }
+
+impl fmt::Debug for Hub {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        f.debug_struct("Hub")
+            .field("surfaces", &self.surfaces)
```

Comment:
is instances the only thing that isn't `Debug`?

Reply:
No, it's also `adapters`.